### PR TITLE
Add Script Command: xkcdpass

### PR DESCRIPTION
**Pull request by crstauf**
https://pypi.org/project/xkcdpass/

### CODE REVIEW

**Review by PitNikola — CHANGES_REQUESTED** [review]
As discussed in Slack and per [new section in guidelines about dependencies](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md#scripts-that-require-installation-of-dependencies), let's add a comment section explicitly stating the dependency and installation instruction. Thanks!

**Comment by crstauf** [comment]
@PitNikola Added dependency comments.

File: commands/communication/xkcdpass.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Dependency: requires xkcdpass (https://pypi.org/project/xkcdpass/).
+# Install via pip: `pip install xkcdpass`
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title xkcdpass
```
Comment (thomaspaulmann) [inline commands/communication/xkcdpass.sh:8]: Let's give it a proper title.
```suggestion
# @raycast.title Generate Passphrase
```

**Review by thomaspaulmann — CHANGES_REQUESTED** [review]
Nice one. What's the difference to #62? Do we need both?

**Comment by crstauf** [comment]
> What's the difference to #62? Do we need both?

Yes.

> There are dozens of random password generators out there that will happily put together a bunch of random characters for you to use as a password. These random passwords are secure, but they're a huge pain to actually remember.
> 
> Random passphrases provide the best combination of memorability and security.

Source: [useapassphrase.com](https://www.useapassphrase.com/#i-get-it-simple-passwords-are-cracked-easily-but-why-should-i-use-a-random-pass-phrase-instead-of-say-ipz2-az8k-0h-)

**Pull request merged**